### PR TITLE
Revert "You can now emag tram crossing signals to disable their motion sensors (#67123)"

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

You can no longer emag crossing signals.

## Why It's Good For The Game

People are just going to ignore the crossing signals because people are going to emag them every single round they can get an emag. It turns the crossing signals into something people ignore for tricking them instead of something they fall for. SS13 players fall for something exactly once, and then after that they assume it's trapped every single round.

In short, letting people emag the crossing signals defeats the point of having the crossing signals in the first place.

## Changelog
:cl:
del: Crossing signals can no longer be emagged.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
When things stop being novel we get a better view of how they'll turn out. I think your concerns are valid but at least give it time to see how things function.
